### PR TITLE
Updated UI

### PR DESCRIPTION
**Pull request by mufida1018**
Hey,
Pls check

### CODE REVIEW

File: lib/screens/remainders_screen.dart [hunk 1]
```diff
@@ -1,4 +1,5 @@
 import 'package:flutter/material.dart';
+import 'package:rflutter_alert/rflutter_alert.dart';
 
 class RemaindersScreen extends StatelessWidget {
```
Comment (ShivamMenda) [inline lib/screens/remainders_screen.dart:4]: Add a key here. Always use key in widget constructors.

File: lib/screens/remainders_screen.dart [hunk 2]
```diff
@@ -7,15 +8,203 @@ class RemaindersScreen extends StatelessWidget {
   Widget build(BuildContext context) {
     return Scaffold(
       appBar: AppBar(
-        title: Text("Remainder\'s"),
+        iconTheme: IconThemeData(
```
Comment (ShivamMenda) [inline lib/screens/remainders_screen.dart:11]: Add const here.

File: lib/screens/remainders_screen.dart [hunk 3]
```diff
@@ -7,15 +8,203 @@ class RemaindersScreen extends StatelessWidget {
...
+          color: Colors.black, //change your color here
+        ),
+         backgroundColor: Colors.transparent,
+        elevation: 0,
+        title: Text("Remainder\'s",style: TextStyle(color: Colors.black,fontWeight: FontWeight.bold),),
```
Comment (ShivamMenda) [inline lib/screens/remainders_screen.dart:16]: Add const here.

File: lib/screens/remainders_screen.dart [hunk 4]
```diff
@@ -7,15 +8,203 @@ class RemaindersScreen extends StatelessWidget {
...
+        actions: [
```
Comment (ShivamMenda) [inline lib/screens/remainders_screen.dart:17]: Add const here

File: lib/screens/remainders_screen.dart [hunk 5]
```diff
@@ -7,15 +8,203 @@ class RemaindersScreen extends StatelessWidget {
...
+                ),
+                 child: Padding(
+                   padding: const EdgeInsets.all(10.0),
+                   child: Column(
+                     mainAxisAlignment: MainAxisAlignment.spaceBetween,
+                     crossAxisAlignment: CrossAxisAlignment.start,
+                     children: [
+                       Column(
+                         children: [
+                           Text("Title"),
```
Comment (ShivamMenda) [inline lib/screens/remainders_screen.dart:55]: Add const here

File: lib/screens/remainders_screen.dart [hunk 6]
```diff
@@ -7,15 +8,203 @@ class RemaindersScreen extends StatelessWidget {
...
+                           Text("Para"),
```
Comment (ShivamMenda) [inline lib/screens/remainders_screen.dart:56]: Add const here too

File: lib/screens/remainders_screen.dart [hunk 7]
```diff
@@ -7,15 +8,203 @@ class RemaindersScreen extends StatelessWidget {
...
+                         ],
+                       ),
+                       Column(
+                         children: [
+                           Container(
+                             height: 2,
+                             width:  MediaQuery.of(context).size.width,
+                             color: Colors.white,
+                           ),
+                           SizedBox(height: 3,),
```
Comment (ShivamMenda) [inline lib/screens/remainders_screen.dart:66]: Add const and increase a little height.

File: lib/screens/remainders_screen.dart [hunk 8]
```diff
@@ -7,15 +8,203 @@ class RemaindersScreen extends StatelessWidget {
...
+                           Row(
+                             mainAxisAlignment: MainAxisAlignment.spaceBetween,
+                             children: [
+                               Text("TODAY"),
```
Comment (ShivamMenda) [inline lib/screens/remainders_screen.dart:70]: Add const here

File: lib/screens/remainders_screen.dart [hunk 9]
```diff
@@ -7,15 +8,203 @@ class RemaindersScreen extends StatelessWidget {
...
+                               Text("6:00 PM"),
```
Comment (ShivamMenda) [inline lib/screens/remainders_screen.dart:71]: Add const here too

File: lib/screens/remainders_screen.dart [hunk 10]
```diff
@@ -7,15 +8,203 @@ class RemaindersScreen extends StatelessWidget {
...
+                           ),
+                         ],
+                       ),
+
+                     ],
+                   ),
+                 ),
+              ),
+            SizedBox(height: 20,),
+            Text("UPCOMING",style:TextStyle(fontWeight: FontWeight.bold)),
```
Comment (ShivamMenda) [inline lib/screens/remainders_screen.dart:82]: Add const and make title italic.

File: lib/screens/remainders_screen.dart (hunk 8 above)
Comment (ShivamMenda) [reply lib/screens/remainders_screen.dart:70]: Make this italic too

File: lib/screens/remainders_screen.dart [hunk 11]
```diff
@@ -7,15 +8,203 @@ class RemaindersScreen extends StatelessWidget {
...
+                        Container(
+                          height: 2,
+                          width:  MediaQuery.of(context).size.width,
+                          color: Colors.white,
+                        ),
+                        SizedBox(height: 3,),
+                        Row(
+                          mainAxisAlignment: MainAxisAlignment.spaceBetween,
+                          children: [
+                            Text("TODAY"),
```
Comment (ShivamMenda) [inline lib/screens/remainders_screen.dart:114]: Add const and make italic

File: lib/screens/remainders_screen.dart [hunk 12]
```diff
@@ -7,15 +8,203 @@ class RemaindersScreen extends StatelessWidget {
...
+                            Text("6:00 PM"),
```
Comment (ShivamMenda) [inline lib/screens/remainders_screen.dart:115]: Add const and make italic

File: lib/screens/remainders_screen.dart [hunk 13]
```diff
@@ -7,15 +8,203 @@ class RemaindersScreen extends StatelessWidget {
...
+                   maxLines: null,
+                   decoration: new InputDecoration.collapsed(
+                       hintText: 'Type here..',
+                   ),
+                 ),
+               ),
+             ],
+           ),
+          ),
+          SizedBox(height: 18,),
```
Comment (ShivamMenda) [inline lib/screens/remainders_screen.dart:171]: Add const

File: lib/screens/remainders_screen.dart [hunk 14]
```diff
@@ -7,15 +8,203 @@ class RemaindersScreen extends StatelessWidget {
...
+        ],
+      ),
+      buttons: [
+        DialogButton(
+          onPressed: () {
+            Navigator.pop(context);
+          },
+          color: Colors.yellow[300],
+          child: Text(
+            "+",
```
Comment (ShivamMenda) [inline lib/screens/remainders_screen.dart:205]: Add const

File: lib/screens/remainders_screen.dart [hunk 15]
```diff
@@ -7,15 +8,203 @@ class RemaindersScreen extends StatelessWidget {
...
+            style: TextStyle(color: Colors.black, fontSize: 35, fontWeight: FontWeight.bold),
```
Comment (ShivamMenda) [inline lib/screens/remainders_screen.dart:206]: Add const

**Review by ShivamMenda — CHANGES_REQUESTED** [review]
@mufida1018 I've suggested some code quality changes. Please do make them. Also please follow the spacing and italics as given in the UI in the readme file. The Today and upcoming title is sticking to the screen. Give some padding there and make text italic wherever necessary.

**Comment by mufida1018** [comment]
Pls check

**Comment by ShivamMenda** [comment]
@mufida1018 will check and get back to you.

**Review by ShivamMenda — APPROVED** [review]
Looks good! Thank you for your contribution!

**Pull request merged**